bouncer/src/repo/core/handler/repo_database_handler_mongo.cpp

```diff
@@ -208,25 +210,28 @@ repo::core::model::RepoBSON createRepoBSON(
 	if (!ignoreExtFile) {
 		if (orgBson.hasFileReference()) {
 			auto ref = orgBson.getBinaryReference();
-
 			auto buffer = blobHandler.readToBuffer(fileservice::DataRef::deserialise(ref));
 			orgBson.initBinaryBuffer(buffer);
 		}
-		else if (orgBson.hasLegacyFileReference()) {
-			std::unordered_map< std::string, std::pair<std::string, std::vector<uint8_t>> > binMap;
-			std::vector<std::pair<std::string, std::string>> extFileList = orgBson.getFileList();
-			for (const auto &pair : extFileList)
-			{
-				auto fileManager = fileservice::FileManager::getManager();
+		else if (orgBson.hasField(REPO_LABEL_OVERSIZED_FILES)) {
+			// The _extRef support has been deprecated, and collections should have been
+			// updated by the 5.4 migration scripts.
+			// In case we have any old documents remaining, this snippet can read the data,
```

Comment:
OK if its gone from .io then I'll remove it completely too!

I have run the 5.4 migration script on the tests database dump, though not pushed yet as this will break the tests for the other branches.

After doing this I realised that the 5.4 migration script didn't write the `size` parameter to the new `_blobRefs`, so I have created an additional migration script which infers it based on the elements, and used that to update the test's dump. We will need to run it against the database for bouncer to load legacy files for real though.